charts/all/openvino/templates/ov-subscription.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
It should not be necessary (and might the gitops operator heartburn) to list the subscription both in the chart and in the subscription section of the clustergroup file. Is this motivated by a problem you had?

Reply:
I had subscription for openvino only in charts and it worked fine. I had problems that openvino was in the _Unknown_ state. I tested it now, works fine, so I moved subscription to values-hub.yaml.